packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1882,8 +1882,7 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
     )
         external
         returns (
-            bool,
-            bytes memory
+            bytes memory _returndata
```

Comment:
If you remove `_returndata` above, probably good to remove it here too.

I think our Solidity style is no longer using these `returns` variable declarations

Reply:
The returned arguments must be named or the geth ABI package will not be able to decode them into a struct

Reply:
Got it! I'd suggest adding a comment in the function about that because it seemed like a code smell to not use the variable that was defined, but I also hear you about not liking to use the variables defined in the `returns` statement.

I also think we should define our Solidity style guide because right now I believe our style is to not use the returns variables, but I hear you on why it's useful to have them in this case. And then if we ever diverge from the style guide we can add comments for why we did.